### DISCUSSION TITLE
指南怎么用

### DISCUSSION THREAD

**Discussion by joey-sam** [iptv-org/community#1162] [Q&A]
我已经按照EPG的步骤实现到guide.xml，且可以通过local:3000来登录
但在iptvnator加载epg时加载错误
后面我把文件上传到github的仓库，使用raw网址加载进去
iptvnator还是会加载错误
请问一下大佬，我哪一步骤错了，急需解决，感谢大佬们

**Comment by freearhey** [comment]
Could you please provide the text of the error?

**Comment by BellezaEmporium** [comment]
-> As Arhey indicated, please provide the error message.
-> Are we talking about a Docker image or a local instance of iptv-org/epg ?
-> As far as I know, we can open the devtools from iptvnator (as it's an Electron app). Look after the "network" tab and see if it gives you a 200 status code when loading your own EPG.
